tests/karate/src/test/java/be/election/castVote.feature

```diff
@@ -27,6 +28,9 @@ Feature: Cast a vote
     * json cast_vote = frontend_buffer.takeTimeout(timeout)
     Then match cast_vote contains deep {jsonrpc: '2.0', id: '#(castVoteId)', result: 0}
 
+  # Testing if after setting up a valid lao, subscribing to it, sending a catchup
+  # creating a valid election setup that casting vote on a lao channel instead of election
```

Comment:
Do you mean :
Testing if after setting up a valid LAO, subscribing to it, sending a catchup and creating a valid election setup, the backend returns an error upon casting a vote on an LAO channel instead of an election one 

Alternatively we could replace "Testing if after setting up a valid LAO, subscribing to it, sending a catchup and creating a valid election setup" by " Testing if after creating an election correctly,"

What do you think ?